app/uk/gov/hmrc/soletraderidentificationfrontend/views/details_not_found_page.scala.html

```diff
@@ -31,25 +32,28 @@
 
 @layout(pageConfig, title = title("details-not-found-page.title")) {
 
-    @h1(messages("details-not-found-page.heading"))
-
+    @p{@h1(messages("details-not-found-page.heading"))}
+    <br>
     @p {@messages("details-not-found-page.line_1")}
 
     @p {@messages("details-not-found-page.line_2")
-        @link(redirectLocation.url, messages("details-not-found-page.link_2"), Some("TryAgain"))
-    }
+        @link(redirectLocation.url,messages("details-not-found-page.link_2"),Some("TryAgain"))
+        }
 
     @p {@messages("details-not-found-page.line_3")}
 
-    @p {@messages("details-not-found-page.line_4.1")
-        <a class='govuk-link' href="@appConfig.ninoTeamUrl" id="ninoTeam" target="_blank">@messages("details-not-found-page.link_4")</a>
-        @messages("details-not-found-page.line_4.2")
-        <strong>@messages("details-not-found-page.line_4.3")</strong>
-        @messages("details-not-found-page.line_4.4")
+        <ul class="govuk-list govuk-list--bullet">
```

Comment:
The GDS Design system does not suggest wrapping <li> elements with paragraphs. If you want to have a greater spacing between the <li> elements use the styling

<ul class="govuk-list govuk-list--bullet govuk-list--spaced">